dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/interceptor/ContextInterceptor.java

```diff
@@ -49,11 +53,7 @@ public class ContextInterceptor implements Interceptor {
 
   private static final String KEY_CURRENT_KEY = "keyCurrentKey";
 
-  private DatabaseInfoProvider databaseInfoProvider;
-
-  public void setDatabaseInfoProvider(DatabaseInfoProvider databaseInfoProvider) {
-    this.databaseInfoProvider = databaseInfoProvider;
-  }
+  private final DatabaseInfoProvider databaseInfoProvider;
```

Comment:
This is to fix a circular dependency between DataBaseInfoProvider and DhisConfigurationProvider